pkg/nixpath/chunker/fastcdc.go

```diff
@@ -0,0 +1,49 @@
+package chunker
+
+import (
+	"errors"
+	"fmt"
+	"io"
+
+	fastcdc "github.com/poolpOrg/go-fastcdc"
+)
+
+func NewFastCDCChunker(r io.Reader) (Chunker, error) { // nolint:ireturn
+	fastcdc.NewChunkerOptions()
+	chunkerOpts := fastcdc.NewChunkerOptions()
+
+	// FUTUREWORK: Test with different chunk sizes
+	chunkerOpts.NormalSize = 64 * 2024
+	chunkerOpts.MinSize = chunkerOpts.NormalSize / 4
+	chunkerOpts.MaxSize = chunkerOpts.NormalSize * 4
```

Comment:
I just used the values here I used in nix-casync initially. This probably should still be refined once we can ingest a bit of data.